sibylapp/view/global_contributions.py

```diff
@@ -9,28 +9,30 @@
 from sibylapp.view.utils import filtering, helpers
 
 
-@st.cache_data(show_spinner="Generating plot...")
-def generate_swarm_plot(contribution_dict):
+@st.cache_data(show_spinner="Generating summary plot...")
+def generate_swarm_plot(eids):
+    contribution_dict = helpers.rename_for_pyreal_vis(contributions.get_contributions(eids))
     swarm_plot(contribution_dict, type="strip")
     return plt.gcf()
 
 
-def view_summary_plot(contribution_dict):
+def view_summary_plot(eids):
     st.pyplot(
-        generate_swarm_plot(helpers.rename_for_pyreal_vis(contribution_dict)),
+        generate_swarm_plot(eids),
         clear_figure=True,
     )
 
 
-def view(all_contributions):
-    sort_by = helpers.show_sort_options(["Total", "Most Increasing", "Most Decreasing"])
-
+def view(eids):
+    sort_by = st.selectbox("Sort order", ["Total", "Most Increasing", "Most Decreasing"])
```

Comment:
This was probably a merge error, but we should be using the new `helpers.show_sort_options` function here